<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1311</width>
    <height>1050</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TACTICAL :: Tracking Algorithm Comparison Testbed for Comprehensive Analysis and Learning</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>570</y>
      <width>560</width>
      <height>310</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>5</number>
    </property>
    <widget class="QWidget" name="tabPreprocess">
     <attribute name="title">
      <string>Preprocess</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tabCorrelation">
     <attribute name="title">
      <string>Correlation</string>
     </attribute>
     <widget class="QLabel" name="labelConstOffset">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Constellation Offset</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxConstOffset">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>20</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelSearchRadius">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Search Radius</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxSearchRadius">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>50</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelSearchLevels">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Search Levels</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxSearchLevels">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>80</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelMinSample">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Min Sample Proportion</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelFovealKernelBaseConstant">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Foveal Kernel Base Constant</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelFovealKernelNumLevels">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Foveal Kernel Number Levels</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelFovealKernelTypeSpacing">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Foveal Kernel Type Spacing</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxDisplayFovealKernel">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>250</y>
        <width>141</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Display Foveal Kernel</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBoxMinSample">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>120</y>
        <width>62</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBoxFovealKernelBaseConstant">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>160</y>
        <width>62</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxFovealKernelNumLevels">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>190</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxFovealKernelTypeSpacing">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>220</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Exponential</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Geometric</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Linear</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="labelCorrErrorThresh">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>10</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Correlation Error Threshold</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelDegeneracyThresh">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>40</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Degeneracy Threshold</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelShapeChangeThresh">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>70</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Shape Change Threshold</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelRejectionRateTimeConstant">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>100</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Rejection Rate Time Constant</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelRejectionRateThresh">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>130</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Rejection Rate Threshold</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelConstExpansionThresh">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>160</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Constellation Expansion Threshold</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelInverseConstellationExpansionThresh">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>190</y>
        <width>211</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Inverse Constellation Expansion Threshold</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelStarOutOfBounds">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>220</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Star Out of Bounds</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelCenterOutOfBounds">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>250</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Center Out of Bounds</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBoxCorrErrorThresh">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>10</y>
        <width>62</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBoxDegeneracyThresh">
      <property name="geometry">
       <rect>
        <x>481</x>
        <y>40</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBoxShapeChangeThresh">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>70</y>
        <width>62</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBoxRejectionRateTimeConstant">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>100</y>
        <width>62</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBoxRejectionRateThresh">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>130</y>
        <width>62</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxConstExpansionThresh">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>160</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBoxInverseConstellationExpansionThresh">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>190</y>
        <width>62</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxStarOutOfBounds">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>220</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxCenterOutOfBounds">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>250</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabOpticalFlow">
     <attribute name="title">
      <string>Optical Flow</string>
     </attribute>
     <widget class="QComboBox" name="comboBoxOFAlgorithm">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>18</y>
        <width>85</width>
        <height>27</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>KLT</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Horn-Schunck</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Farneback</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Algorithm</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxOFShowFeatures">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>240</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Show Features?</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabBlob">
     <attribute name="title">
      <string>Blob</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tabNavigation">
     <attribute name="title">
      <string>Navigation</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Debug Messages</string>
     </attribute>
     <widget class="QPlainTextEdit" name="plainTextEditTrace">
      <property name="geometry">
       <rect>
        <x>7</x>
        <y>10</y>
        <width>541</width>
        <height>251</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>642</width>
      <height>482</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonTrack">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>510</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Track</string>
    </property>
   </widget>
   <widget class="QScrollBar" name="imageScrollBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>651</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxFitToWindow">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Fit to window</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_2">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>0</y>
      <width>642</width>
      <height>482</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>570</y>
      <width>560</width>
      <height>310</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>4</number>
    </property>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Smoothing</string>
     </attribute>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>34</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Filter</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>30</y>
        <width>131</width>
        <height>27</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Mean Filter :: Arithmetic</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mean Filter :: Geometric</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mean Filter :: Contraharmonic</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mean Filter :: Harmonic</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Order Statistics :: Median</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Order Statistics :: Max</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Order Statistics :: Min</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Order Statistics :: Midpoint</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Order Statistics :: Alpha Trimmed</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Adaptive Local Noise Reduction Filter</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Adaptive Median Filter</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Mask Size</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>90</y>
        <width>85</width>
        <height>27</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>1x1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3x3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5x5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>7x7</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Sharpening</string>
     </attribute>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Algorithm</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_3">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>40</y>
        <width>85</width>
        <height>27</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Highboost Laplacian</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Gradient</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>Reduce Graylevels</string>
     </attribute>
     <widget class="QComboBox" name="comboBoxReduceGraylevels">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>36</y>
        <width>85</width>
        <height>27</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>128</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>64</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>32</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>16</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>8</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>161</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Number of Gray levels</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_7">
     <attribute name="title">
      <string>Histogram Equalization</string>
     </attribute>
     <widget class="QCheckBox" name="checkBoxHistogramEqualization">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>241</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Apply histogram equalization?</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_8">
     <attribute name="title">
      <string>Noise</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QComboBox" name="comboBoxSecondWindow">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>526</y>
      <width>191</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Blank</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Show Processed Image</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Show Previous Frame</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>531</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Second Window</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxSwapRedBlue">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>530</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Swap red and blue?</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1311</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Open_Sequence"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
   </widget>
   <widget class="QMenu" name="menuDataset_Actions">
    <property name="title">
     <string>Dataset Actions</string>
    </property>
    <addaction name="actionApply_to_Entire_Dataset"/>
    <addaction name="actionExport_Dataset"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuDataset_Actions"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Open_Sequence">
   <property name="text">
    <string>&amp;Open Sequence or Movie</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionEvaluation">
   <property name="text">
    <string>Evaluation</string>
   </property>
  </action>
  <action name="actionNavigation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Navigation</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionApply_to_Entire_Dataset">
   <property name="text">
    <string>Apply to Entire Dataset</string>
   </property>
  </action>
  <action name="actionExport_Dataset">
   <property name="text">
    <string>Export Dataset</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
